doc/APIGuide.md

```diff
@@ -57,28 +57,28 @@ model.getMassCenter(state); // Need to know pose of model.
+2. Discrete state variables are system unknowns that may or may not be governed by differential or algebraic equations, and include external inputs and controls. For example, when the actuation (force or tension that an actuator produces) of an Actuator is overridden, the override-actuation is a discrete state variable and must be supplied.
+3. Cache variables allow saving of (state-dependent) calculations that may be used multiple times, to improve computational efficiency. For example, computing the path length of a muscle with several intermediate points and wrapping over obstacles is expensive. Therefore, a path component itself caches its length so that the muscle (or any other component) that needs the length in its computations does not reevaluate it unnecessarily. The validity of the path length cache variable is dependent on the generalized coordinates--as long as these generalized coordinates do not change, the path length is valid. OpenSim employs categories of dependency known as realization Stages (see Simbody User Guide, Section 2.4 for details) to manage the validity of cache variables. Any changes to the generalized coordinates invalidate (and wipe out) any cache variables associated with the Position Stage and all other subsequent (Velocity, Dynamics, Acceleration) stages. OpenSim automatically invalidates cache variables to support the correctness of simulations. Manual cache management is highly prone to errors leading to “stale” variables and potentially incorrect physics. Sadly, while results may appear plausible they are completely invalid and sometimes impossible to detect and debug. If you need to cache for performance, use OpenSim’s facilities.
 
 ### Models are composed of Components {#modelscomposed}
 A Component encapsulates (all or part of the) system dynamics (of a model or element of a model) and provides (computes) values of interest. A component adds its dynamics and allocates necessary resources (state, cache, and other variables). It defines parameters (see Properties below) that specify its behavior/function and defines dependencies on any other Components (see Sockets below) via its Inputs and Outputs.
 
-Components of a [Model](@ref OpenSim::Model) form a rooted directed tree topology of ownership (Components own and know about their subcomponents not the other way around). A root component (the top level Model) contains all the necessary Components to define the System. Components in the tree are uniquely identified by their full path name from the root (like a file path)
+Components of a [Model](@ref OpenSim::Model) form a rooted directed tree topology of ownership (Components own and know about their descendents). A root component (the top level Model) contains all the necessary Components to define the System. Components in the tree are uniquely identified by their full path name from the root (like a file path)
```

Comment:
I prefer subcomponents to descendents.

Reply:
@aseth1 the previous sentence try to paint a mental picture of a tree (well understood abstraction) so I'd expect this sentence to describe the tree using the standard 'tree' terminology (children, descendents, etc.) not using subcomponents.

Reply:
The statement in brackets is about Components owning ###,  and we call those owned components "subcomponents" not descendants. Another/additional statement that subcomponents are the descendants in the tree would also be OK.

Reply:
There are two points.
1. Components own their subcomponents.
2. Components are nodes in the tree and subcomponents are all the descendants that branch from a node in the ownership tree.